pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -1063,12 +1065,7 @@ pub mod pallet {
 			Self::ensure_unlocked(&avatar_id)?;
 			ensure!(Preparation::<T>::contains_key(avatar_id), Error::<T>::NotPrepared);
 
-			Self::do_transfer_avatar(
-				&player,
-				&Self::technical_account_id(),
-				&avatar.season_id,
-				&avatar_id,
-			)?;
+			Self::try_remove_avatar_ownership_from(&player, &avatar.season_id, &avatar_id)?;
```

Comment:
It feels a little bit strange to put them into the limbus with no owner.

Reply:
Well, its is the simplest solution without adding new additional storage, and allows unlimited avatar locking just like we wanted